app/src/libs/quest.ts

```diff
@@ -1146,6 +1156,12 @@ export const isAvailableUserQuests = (
       );
     });
 
+  // Daily quests should be available even if previously completed (they reset daily)
+  const completedCheck =
+    questAndUserQuestInfo.questType === "daily" ||
+    !questAndUserQuestInfo.completed ||
+    questAndUserQuestInfo.completed === 0;
```

Comment:
This conflicts with `eventCompletedCheck`, no? i.e. some quests we allow multiple completes of, but this would prevent any non-daily multiple completes